service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -1,32 +1,25 @@
 package uk.gov.hmcts.reform.fpl.utils;
 
-import lombok.RequiredArgsConstructor;
 import org.apache.commons.lang3.StringUtils;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.model.HearingBooking;
-import uk.gov.hmcts.reform.fpl.model.common.Element;
-import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
 
 import java.time.format.FormatStyle;
-import java.util.List;
 import java.util.stream.Stream;
 
 import static java.util.stream.Collectors.joining;
 import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
-import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
-@Component
-@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class EmailNotificationHelper {
 
-    private final HearingBookingService hearingBookingService;
+    private EmailNotificationHelper() {
+    }
 
-    public String buildSubjectLine(final CaseData caseData) {
+    public static String buildSubjectLine(final CaseData caseData) {
```

Comment:
Can we refactor caseData out of this, `buildSubjectLineWithHearingBookingDateSuffix`, and `buildSubjectLineWithoutHearingBookingDateSuffix`

Reply:
can indeed :)